src/sage/graphs/graph_decompositions/tree_decomposition.pyx

```diff
@@ -779,6 +780,116 @@ def treewidth(g, k=None, kmin=None, certificate=False, algorithm=None):
 # Treelength
 #
 
+def length_of_tree_decomposition(G, T, check=True):
+    r"""
+    Return the length of the tree decomposition `T` of `G`.
+
+    The *length* of a tree decomposition, as proposed in [DG2006]_, is the
+    maximum *diameter* in `G` of its bags, where the diameter of a bag `X_i` is
+    the largest distance in `G` between the vertices in `X_i` (i.e., `\max_{u, v
+    \in X_i} dist_G(u, v)`). See the documentation of the
+    :mod:`~sage.graphs.graph_decompositions.tree_decomposition` module for more
```

Comment:
I think "dist" is a quite common term. 

We may define a latex macro definition to `sage.misc.latex_macros.latex_macros" so that we may use "\dist" conveniently. The list of the predefined macros is short and I think we can make more use of it.

I suggest  "\newcommand{\dist}{\mathrm{dist}}". Hopefully `\dist` is not predefined in tex or latex.

Reply:
Good idea, done.

Actually, it's used only 3 times in `src/sage/graphs/`... I have not checked in other modules.